### ISSUE TITLE
Add remote navigation support

### ISSUE THREAD

**Issue by cutterchop** [CarlosEsco/Neko#1559]
### Describe your suggested feature

I was able to use and read on my android tv with minor issues but I had to use a mouse , the remote navigation is hard so you can fix the remote mapping for tv users, I'm not sure how much work this is but it's a opening for you to have a extra unique feature in your fork by opening up for Android tv support adding extra accessibility so please consider and thanks 🙏

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the newest version **[Latest](https://github.com/CarlosEsco/Neko/releases/latest)**.
- [X] I have checked through the app settings for my feature.
- [X] I will fill out all of the requested information in this form.

**Comment by cutterchop** [comment]
> Can you provide some of the pain points?

Yes, I was attempting to take a video and test the application's functionality. However, I encountered an issue where the selector appeared to be stuck on the filter button and would not move at all. It seems that I was testing Tachi J2K at the time, but I was under the impression that Neko shared the same user interface. When I tested the application on J2K, I was able to move around in explore and scroll while reading a manga. However, there was an issue where settings were not accessible. I believe J2K may have attempted to implement some form of navigation, but it was later discontinued. It would be helpful if you could port the implemented navigation from J2K, and then after i can test for the pain points from there

**Comment by CarlosEsco** [comment]
It would be easier to list out the pain points vs me porting the j2k stuff because it's not a 1 to 1 with j2k anymore

**Comment by cutterchop** [comment]
> It would be easier to list out the pain points vs me porting the j2k stuff because it's not a 1 to 1 with j2k anymore

I see well, by launching the app and trying to navigate it appears that it does not respond at all except by clicking in which opens up the option that lets u sort the display and that is it, so it likely needs navigation implementation

**Comment by CarlosEsco** [comment]
> > It would be easier to list out the pain points vs me porting the j2k stuff because it's not a 1 to 1 with j2k anymore
> 
> I see well, by launching the app and trying to navigate it appears that it does not respond at all except by clicking in which opens up the option that lets u sort the display and that is it, so it likely needs navigation implementation, tho I did say its working because I was under the consideration that the same UI is shared 
> 
> https://developer.android.com/training/tv/start/navigation this might help if u are willing

I assume you're using the dpad?

**Comment by cutterchop** [comment]
> I assume you're using the dpad?

yup -  Hisense android tv remote ( example image on bottom ) 

![image](https://github.com/CarlosEsco/Neko/assets/31880937/151e0da7-d394-4e31-9b34-8e3fbd47dd34)

**Comment by cutterchop** [comment]
bump?

**Comment by Efreak** [comment]
*in the meantime*, you can use [key mapper](https://github.com/keymapperorg/KeyMapper) to remap keys while specific apps are running. This can do things like tap on certain areas of the screen; I use it to control my music with Bluetooth remote's dpad while the Tachiyomi/neko/FBReader/browser are open and Poweramp is playing music.
